css-structure/css-structure_selectors-and-cascading/css/styles.css

```diff
@@ -49,92 +49,92 @@ nav li {
 /* -------------------------------------------------------- */
 
 /* 1. These styles should apply to the headline */
-FIX_ME {
+header {
   outline: 2px dashed red;
   font-size: 2rem;
   margin: 0 2rem 0 0;
   color: #2673bb;
 }
 
 /* 2. These styles should apply to all links */
-FIX_ME {
+a {
   outline: 2px dashed red;
   color: #ff00e6;
 }
 
 /* 3. These styles should apply to all external links (you can use the attribute selector) */
-FIX_ME {
+.text a {
```

Comment:
This will still select all links, no matter if external or internal.
you might want to use an attribute selector here: `a[href^="http"]` 
This will select all links **starting with** `http`
check out: https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors